PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+## Description
+
+`Delete me: Describe the changes in this PR. Please explicitly note any breaking changes.`
+
+**Jira**: <https://jira.macuject.com/[ID]>
+
+## Screenshots / Videos
+
+`Delete me: If applicable, add screenshots to help explain this PR's changes. If not needed delete this entire section.`
+
+| Type         | Before | After |
+| ------------ | ------ | ----- |
+| Wide screen  | IMG    | IMG   |
+| Small screen | IMG    | IMG   |
+
+## Data
+
+`Delete me: Does this PR require database, data migration or data flow change? Describe the changes required and testing undertaken. If not needed delete this entire section.`
+
+## Security
+
+`Delete me: Does this PR modify any aspects of our security posture? If so, please describe the security changes, discussions had with the Macuject Information Security Officer, and the testing undertaken. If not needed delete this entire section.`
+
+## Impact Assessment
```

Comment:
Does this applicable only to production repos?
This may not be applicable to ML R&D reposts and pre-production repos I assume @bradleybeddoes?

Reply:
I agree; some sections will be easy deletes. There is an ask to GitHub to support multiple templates, of which one must be chosen @gihants. When that ships, we'll appropriately customise for the web, ML, and platform.

Reply:
Great, thanks @bradleybeddoes!